app/admin/todo_list.rb

```diff
@@ -37,7 +37,13 @@
       f.input :title
       f.input :deadline, as: :datepicker, datepicker_options: { min_date: Time.now.to_date }
     end
-
+    f.inputs 'Assignees' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.